src/connector/sqlite.rs

```diff
@@ -53,6 +55,8 @@ impl TryFrom<&str> for SqliteParams {
         } else {
             let mut connection_limit = None;
             let mut socket_timeout = None;
+            let mut max_connection_lifetime = None;
+            let mut max_idle_connection_lifetime = None;
```

Comment:
I think this is a good default for SQLite. See in https://github.com/prisma/prisma-engines/blob/master/query-engine/connectors/sql-query-connector/src/database/sqlite.rs#L51 we have a different value, but I'd remove it completely and let the default be infinite (SQLite doesn't hold a network connection anyhow).